src/Rasa.Game/Data/EntityType.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Rasa.Data
+{
+    public enum EntityType
+    {
+        Character   = 0,
```

Comment:
Make sure we start the types from 1, so there are always some bits in the type in the guid.